packages/ignite/src/commands/plugin.js

```diff
@@ -0,0 +1,145 @@
+// @cliDescription Manages ignite plugins
+// @cliAlias p
+// ----------------------------------------------------------------------------
+
+const exitCodes = require('../lib/exitCodes')
+
+/**
+ * Does a walkthrough of questions and returns the answers as an object.
+ * 
+ * @param {Object} context The gluegun context.
+ * @returns {Object} The answers.
+ */
+const walkthrough = async (context) => {
+  const minOptions = { template: 'No', command: 'No' }
+  const maxOptions = { template: 'Yes', command: 'Yes' }
+  if (context.parameters.options.min) { return minOptions }
+  if (context.parameters.options.max) { return maxOptions }
+
+  // Okay, we'll ask one by one, fine
+  return await context.prompt.ask([
+    {
+      name: 'template',
+      message: 'Would you like to generate an example component?',
+      type: 'list',
+      choices: ['No', 'Yes']
+    },
+    {
+      name: 'command',
+      message: 'Would you like to generate an example command/generator?',
+      type: 'list',
+      choices: ['No', 'Yes']
+    }
+  ])
+}
+
+/**
+ * Checks whether a plugin name was given and errors if not.
+ *
+ * @param {Object} context The gluegun context.
+ * @returns {Boolean}
+ */
+const checkName = (pluginName, context) => {
+  const { strings, print, runtime, parameters } = context
+  
+  if (strings.isBlank(pluginName)) {
+    print.info(`${runtime.brand} plugin new <${runtime.brand}-plugin-name>\n`)
```

Comment:
`runtime.brand` is 100% correct.  We use the `ignite` everywhere.  I wouldn't worry about using `runtime.brand` at all in ignite.

SEE, I DON"T HATE BRANDS JAMON.